tests/messages_tests/tests.py

```diff
@@ -185,3 +187,20 @@ def test_mismatching_length(self):
         )
         with self.assertRaisesMessage(AssertionError, msg):
             self.assertMessages(response, [])
+
+    def test_method_frames_ignored_by_unittest(self):
+        response = FakeResponse()
+        add_message(response.wsgi_request, constants.INFO, "INFO message.")
+        try:
+            self.assertMessages(response, [])
```

Comment:
For the sake of simplicity, I would suggest to do the following instead, to not distract the reader with a call (`add_message`) that is not strictly necessary for the correctness of the test:
```suggestion
        try:
            self.assertMessages(response, [object()])
```